### DISCUSSION TITLE
React-location docs vanished

### DISCUSSION THREAD

**Discussion by borie88** [TanStack/router#672] [General]
The old URL points to Tanstack/router documentation now and there doesn't seem to be any way to access the old react-location docs aside from using an internet archive. Could the old documentation be put back online somewhere?

Also, has an upgrade guide been written somewhere or is it safe to say that tanstack/router is so different than react-location that rewriting everything would be easier?

**Comment by tordans** [comment]
It looks like the only way to see the old docs is from the source code of the main branch at https://github.com/TanStack/router/tree/main/docs

Maybe someone can fork and publish a GH Page with the docs?

**Comment by tordans** [comment]
Actually, this is a duplicate to https://github.com/TanStack/router/discussions/607 which has an answer by Tanner on what is planned.

**Reply by tordans** [reply]
@borie88 could close this, so the other answer gets boosted?
